### PR TITLE
Adding DYJetsToMuMu_HT_mll105to160 cards

### PR DESCRIPTION
**Pull request by UAEDF-tomc**
DYJetsToMuMu, HT bin 105 to 160, pre2017 setup

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat]: please remove the banner

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm
+#import model sm-ckm_no_b_mass
+#customize_model —save=ckm_no_b_mass
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat]: please implement https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by UAEDF-tomc** [comment]
Hi @perrozzi the cards should now follow your recommendations. Note that this is a pre2017 request, so the pdf settings are unchanged.

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import model sm
+define l+ =  mu+
+define l- =  mu-
+define vl = vm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/DYJetsToMuMu_HT_mll105to160/DYJetsToMuMu_HT_mll105to160_proc_card.dat]: neutrinos are not used, please remove

**Pull request merged**